### PR TITLE
Contact form and Footer 

### PR DESCRIPTION
**Pull request by tumainimaganiko**
In this project I did:

- [ ] Add Contact form
- [ ] Add footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,5 +184,36 @@ <h2>About me</h2>
             </ul>
         </div>
     </section>
+    <section class="contact-form">
+        <h2>
+            I'm always interested in hearing about new projects,
+             so if you'd like to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/xeqwlyqy">
```
Comment (pinkmoon25) [inline index.html]: Please make sure the form attribute contains, `method="POST"` so that the form posts the data successfully

File: styles.css [hunk 2]
```diff
@@ -346,3 +356,77 @@
...
+  letter-spacing: 0.03em;
+  text-align: center;
+  padding: 12px;
+  margin: 0 24px;
+  margin-top: 24px;
+  background-color: #ff6b00;
+  color: #fff;
+  border: none;
+  margin-bottom: 78px;
+}
```
Comment (pinkmoon25) [inline styles.css]: Please make sure to implement the button interaction such as hover and active effect as given in the figma design.

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @tumainimaganiko @Mahamadou-Nouridine & @shihabrafio  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Linters :heavy_check_mark: 
GitHub flow :heavy_check_mark: 
Readme looks good :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (tumainimaganiko) [reply index.html]: I have added post method to the form element

Comment (tumainimaganiko) [reply index.html]: I have added the button interactions as required

File: styles.css [hunk 3]
```diff
@@ -346,3 +356,96 @@
...
+
+#submit:hover {
+  background-color: #dbd8d7;
+  color: #e05304;
+}
+
+#submit:active {
+  background-color: #e05304;
+  color: #fff;
+}
```
Comment (KanzaTahreem) [inline styles.css]: Good job implementing the button interactions 🥇 However, please make sure to follow the Figma design guidelines while adding interactions, as suggested by the reviewer [here](https://github.com/tumainimaganiko/Portfolio/pull/3#discussion_r1151001516) ⚒
- [ ] I noticed that you are using  `background-color: #dbd8d7;` whereas in Figma template it is `#FFF5E1`. Also please make sure to implement the border for `hover` interaction. See images below for more clarity 👇
- [ ]  Similarly, please fix the color. In Figma template it is `#F55800` for `hover` and `#F4F1F0` for `active`

Your design | Figma template
---| ---|
image | image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @tumainimaganiko 👋

Good job implementing the changes suggested by the reviewer 🥇🥇 
**_However, there are few things that still need to be fixed. But you are almost there!_** 💪 🏁

so_close

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Well done following the given Figma design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

However, there is always room for improvement.

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @tumainimaganiko, @Mahamadou-Nouridine and @shihabrafio, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Highlights
Great collaboration ✅

## Optional suggestions
[OPTIONAL] Kindly write your pull request summary in the past tense because the purpose of the pull requests summary is to tell others about changes you've pushed to the branch.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**